man/profile_compare-methods.Rd

```diff
@@ -139,11 +139,20 @@ s$p2[1:na.required] <- NA
 f.check.NA <- function(i) length(which(is.na(i$p1) | is.na(i$p2))) / nrow(i) == 1
 missing.too.much.data.idx <- which(profileApply(s, f.check.NA))
 
-# remove bad profiles and try again: works
-s.no.na <- profile_compare(s[-missing.too.much.data.idx, ], vars=c('p1','p2'), 
-max_d=100, k=0, plot.depth.matrix=TRUE)
 
 
+# remove bad profiles and try again: works
```

Comment:
Nice catch. I'll review the current logic, wouldn't be surprised if it has missed something.